src/models/calendar.js

```diff
@@ -54,6 +54,8 @@ const getDefaultCalendarObject = (props = {}) => Object.assign({}, {
 	calendarObjects: [],
 	// Time-ranges that have already been fetched for this calendar
 	fetchedTimeRanges: [],
+	// Scheduling transparency
+	transparency: '',
```

Comment:
```suggestion
	transparency: 'opaque',
```

for the correct default?